MateuszSapala/240.Search_a_2D_Matrix_II/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
file should be ignored by gitignore

Reply:
Without this file you won't run project. If this bothers you so much, I will commit only sollution code files from now on